src/litserve/loops/continuous_batching_loop.py

```diff
@@ -135,8 +135,8 @@ async def prefill(
             if self.has_capacity(lit_api):
                 response_queue_id, uid, _, input = request
                 logger.debug(f"New request: {uid}, {input}")
-                self.add_request(uid, input, lit_api, lit_spec)
                 self.response_queue_ids[uid] = response_queue_id
+                self.add_request(uid, input, lit_api, lit_spec)
```

Comment:
why do we need to change the order here?

Reply:
we don't. while finding the issue i was changing few stuff and missed to revert this.